k8s-do/k8s/ingress-tls/README.md

```diff
@@ -0,0 +1,73 @@
+# ingress-tls
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1.6.1](https://img.shields.io/badge/AppVersion-v1.6.1-informational?style=flat-square)
+
+ingress-tls installs and configures cert-manager, an ingress controller and a Let's Encrypt Issuer for automating TLS Certificates on AsyncAPI K8s services.
```

Comment:
```suggestion
`ingress-tls` installs and configures `cert-manager`, an `ingress` controller and a Let's Encrypt Issuer for automating TLS Certificates on AsyncAPI K8s services.
```

Reply:
This is autogenerated using https://github.com/norwoodj/helm-docs. I can change the Chart.yaml instead.

Reply:
Ok, so please don't change that :)

Reply:
Done!

Reply:
It looks right now! description is formatted and then docs are generated properly.